inc/functions/options.php

```diff
@@ -197,24 +197,27 @@ function rocket_get_ignored_parameters() {
  * @since 2.4.1 Auto-exclude WordPress REST API.
  * @since 2.0
  *
+ * @param bool $force Force the static uris to be reverted to null.
+ *
  * @return string A pipe separated list of rejected uri.
  */
-function get_rocket_cache_reject_uri() { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
+function get_rocket_cache_reject_uri( $force = false ) { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
 	static $uris;
+	$wp_rewrite = $GLOBALS['wp_rewrite'];
```

Comment:
```suggestion
	global $wp_rewrite;
```

`$wp_rewrite`  is a global set by WordPress directly, we can access it that way